pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7784300?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7784300?buildTab=artifacts#/) on master @ [db71e045cc3f2937bf20170d254dd7e54a620ffb](https://github.com/cockroachdb/cockroach/commits/db71e045cc3f2937bf20170d254dd7e54a620ffb):


```
W221201 07:18:20.047332 1732 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 264  exiting heartbeat loop
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265  failed to load protected timestamp records: failed to read records: protectedts-GetRecords: context canceled
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +(1) attached stack trace
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  -- stack trace:
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:376
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | [...repeated from below...]
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +Wraps: (2) failed to read records
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +Wraps: (3) attached stack trace
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  -- stack trace:
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:368
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:322
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | runtime.goexit
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +Wraps: (4) protectedts-GetRecords
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +Wraps: (5) context canceled
E221201 07:18:20.047551 2092 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=818573145815810049] 265 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
I221201 07:18:20.047767 1903 jobs/registry.go:1379  [nsql1] 266  AUTO SPAN CONFIG RECONCILIATION job 818573145815810049: stepping through state succeeded
I221201 07:18:20.047953 1805 sql/stats/automatic_stats.go:509  [nsql1] 267  quiescing auto stats refresher
W221201 07:18:20.048019 1811 jobs/registry.go:959  [nsql1] 268  canceling all adopted jobs due to stopper quiescing
E221201 07:18:20.048380 1903 jobs/registry.go:831  [-] 269  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.13s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
/cc @cockroachdb/jobs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7816683?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7816683?buildTab=artifacts#/) on master @ [ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410](https://github.com/cockroachdb/cockroach/commits/ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410):


```
W221203 07:53:36.611346 1570 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 243  error consuming TenantSettings RPC: grpc: node unavailable; try another peer [code 2/Unknown]
I221203 07:53:36.611449 1730 sql/stats/automatic_stats.go:509  [nsql1] 244  quiescing auto stats refresher
W221203 07:53:36.611506 1736 jobs/registry.go:953  [nsql1] 245  canceling all adopted jobs due to stopper quiescing
I221203 07:53:36.611556 1773 jobs/registry.go:1373  [nsql1] 246  AUTO SPAN CONFIG RECONCILIATION job 819146312486027265: stepping through state succeeded
E221203 07:53:36.611619 1773 jobs/registry.go:825  [-] 247  error getting live session: node unavailable; try another peer
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248  failed to load protected timestamp records: failed to read records: protectedts-GetRecords: context canceled
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +(1) attached stack trace
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  -- stack trace:
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:376
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | [...repeated from below...]
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +Wraps: (2) failed to read records
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +Wraps: (3) attached stack trace
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  -- stack trace:
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:368
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:322
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | runtime.goexit
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +Wraps: (4) protectedts-GetRecords
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +Wraps: (5) context canceled
E221203 07:53:36.611894 1945 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819146312486027265] 248 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.16s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7826400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7826400?buildTab=artifacts#/) on master @ [859e1e1f26d79af43cce16519f9ba984baaa495c](https://github.com/cockroachdb/cockroach/commits/859e1e1f26d79af43cce16519f9ba984baaa495c):


```
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:368
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:322
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | runtime.goexit
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +Wraps: (4) protectedts-GetRecords
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +Wraps: (5) context canceled
E221204 07:44:51.601908 705 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=819427704846811137] 235 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
W221204 07:44:51.602211 356 sql/sqlliveness/slinstance/slinstance.go:312  [n1] 236  exiting heartbeat loop
W221204 07:44:51.602267 356 sql/sqlliveness/slinstance/slinstance.go:297  [n1] 237  exiting heartbeat loop
E221204 07:44:51.602542 373 jobs/registry.go:869  [n1] 238  error expiring job sessions: expire-sessions: context canceled
E221204 07:44:51.602606 373 jobs/registry.go:877  [n1] 239  failed to serve pause and cancel requests: context canceled
W221204 07:44:51.602631 373 jobs/registry.go:953  [n1] 240  canceling all adopted jobs due to stopper quiescing
I221204 07:44:51.602933 732 jobs/registry.go:1373  [n1] 241  AUTO SPAN CONFIG RECONCILIATION job 819427704846811137: stepping through state succeeded
E221204 07:44:51.603024 732 jobs/registry.go:825  [n1] 242  error getting live session: node unavailable; try another peer
W221204 07:44:51.602938 1635 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 243  error consuming TenantSettings RPC: grpc: context canceled [code 1/Canceled]
W221204 07:44:51.603323 1735 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 244  exiting heartbeat loop
W221204 07:44:51.603359 1735 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 245  exiting heartbeat loop
W221204 07:44:51.603420 1634 ccl/kvccl/kvtenantccl/connector.go:231  [nsql1,tenant-connector] 246  error consuming GossipSubscription RPC: grpc: error reading from server: read tcp 127.0.0.1:59566->127.0.0.1:39341: use of closed network connection [code 14/Unavailable]
E221204 07:44:51.603609 2101 kv/kvserver/protectedts/ptreconcile/reconciler.go:155  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819427707349991425] 247  failed to reconcile protected timestamp with id a69dfdad-aad3-431d-a329-dc830225fdd4: txn already encountered an error; cannot be used anymore (previous err: failed to send RPC: sending to all replicas failed; last error: rpc error: code = Unknown desc = node unavailable; try another peer)
I221204 07:44:51.603680 1800 sql/stats/automatic_stats.go:509  [nsql1] 249  quiescing auto stats refresher
W221204 07:44:51.603668 1806 jobs/registry.go:953  [nsql1] 248  canceling all adopted jobs due to stopper quiescing
I221204 07:44:51.603671 2042 jobs/registry.go:1373  [nsql1] 250  AUTO SPAN CONFIG RECONCILIATION job 819427707349991425: stepping through state succeeded
E221204 07:44:51.603779 2042 jobs/registry.go:825  [-] 251  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.11s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7853057?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7853057?buildTab=artifacts#/) on master @ [146556e19f5e4fdc8c3e6a623b280cc33aee4d18](https://github.com/cockroachdb/cockroach/commits/146556e19f5e4fdc8c3e6a623b280cc33aee4d18):


```
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  -- stack trace:
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetMetadata
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:301
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | [...repeated from below...]
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +Wraps: (2) failed to read metadata
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +Wraps: (3) attached stack trace
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  -- stack trace:
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).queryInternalBuffered
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:588
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowExWithCols
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:640
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowEx
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:626
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetMetadata
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:297
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:318
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | runtime.goexit
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +Wraps: (4) protectedts-GetMetadata
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +Wraps: (5) context canceled
E221206 06:53:04.682997 2010 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=819983758193950721] 254 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
E221206 06:53:04.683263 1847 jobs/registry.go:825  [-] 255  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.20s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7868411?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7868411?buildTab=artifacts#/) on master @ [8165e3974c10e88b6ae11c6255872ea16f3a67e3](https://github.com/cockroachdb/cockroach/commits/8165e3974c10e88b6ae11c6255872ea16f3a67e3):


```
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).queryInternalBuffered
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:588
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowExWithCols
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:640
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).QueryRowEx
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:626
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetMetadata
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:297
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:318
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | runtime.goexit
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +Wraps: (4) protectedts-GetMetadata
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +Wraps: (5) context canceled
E221207 07:50:25.024835 1916 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820278146688385025] 238 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
W221207 07:50:25.025220 1651 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 239  exiting heartbeat loop
W221207 07:50:25.027277 1651 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 242  exiting heartbeat loop
I221207 07:50:25.025221 1711 jobs/registry.go:1373  [nsql1] 240  AUTO SPAN CONFIG RECONCILIATION job 820278146688385025: stepping through state succeeded
I221207 07:50:25.023673 645 jobs/registry.go:1373  [n1] 232  AUTO SPAN CONFIG RECONCILIATION job 820278145049886721: stepping through state succeeded
E221207 07:50:25.025910 516 jobs/registry.go:869  [n1] 241  error expiring job sessions: expire-sessions: context canceled
E221207 07:50:25.027484 1711 jobs/registry.go:825  [-] 243  error getting live session: node unavailable; try another peer
E221207 07:50:25.027508 516 jobs/registry.go:877  [n1] 244  failed to serve pause and cancel requests: context canceled
W221207 07:50:25.027577 516 jobs/registry.go:953  [n1] 245  canceling all adopted jobs due to stopper quiescing
E221207 07:50:25.027993 645 jobs/registry.go:825  [n1] 246  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.14s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7880878?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7880878?buildTab=artifacts#/) on master @ [ec095bc2fdbe4e518b076db20e4920fab67222bf](https://github.com/cockroachdb/cockroach/commits/ec095bc2fdbe4e518b076db20e4920fab67222bf):


```
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +Wraps: (2) failed to read records
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +Wraps: (3) attached stack trace
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  -- stack trace:
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:368
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:322
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | runtime.goexit
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +Wraps: (4) protectedts-GetRecords
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +Wraps: (5) context canceled
E221208 07:16:01.113351 788 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [n1,job=AUTO SPAN CONFIG RECONCILIATION id=820554497947303937] 239 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
I221208 07:16:01.113987 1730 sql/stats/automatic_stats.go:509  [nsql1] 240  quiescing auto stats refresher
I221208 07:16:01.113996 609 jobs/registry.go:1373  [n1] 241  AUTO SPAN CONFIG RECONCILIATION job 820554497947303937: stepping through state succeeded
E221208 07:16:01.114082 609 jobs/registry.go:825  [n1] 242  error getting live session: node unavailable; try another peer
W221208 07:16:01.114177 1736 jobs/registry.go:953  [nsql1] 243  canceling all adopted jobs due to stopper quiescing
W221208 07:16:01.114686 1652 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 244  exiting heartbeat loop
W221208 07:16:01.114591 1586 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 246  error consuming TenantSettings RPC: grpc: error reading from server: read tcp 127.0.0.1:36850->127.0.0.1:42909: use of closed network connection [code 14/Unavailable]
I221208 07:16:01.114681 1869 jobs/registry.go:1373  [nsql1] 245  AUTO SPAN CONFIG RECONCILIATION job 820554499480453121: stepping through state succeeded
W221208 07:16:01.114715 1652 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 247  exiting heartbeat loop
W221208 07:16:01.114625 1585 ccl/kvccl/kvtenantccl/connector.go:233  [nsql1,tenant-connector] 248  error consuming GossipSubscription RPC: grpc: error reading from server: read tcp 127.0.0.1:36850->127.0.0.1:42909: use of closed network connection [code 14/Unavailable]
E221208 07:16:01.114782 1869 jobs/registry.go:825  [-] 249  error getting live session: node unavailable; try another peer
E221208 07:16:01.115184 1987 kv/kvserver/protectedts/ptreconcile/reconciler.go:155  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=820554499480453121] 250  failed to reconcile protected timestamp with id 742c2940-5138-4430-81e5-c9be6b66272a: txn already encountered an error; cannot be used anymore (previous err: failed to send RPC: sending to all replicas failed; last error: rpc error: code = Unknown desc = node unavailable; try another peer)
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.14s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910229?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910229?buildTab=artifacts#/) on master @ [f2b00e8039af6ea8887ec124dad8daf19da6fbf1](https://github.com/cockroachdb/cockroach/commits/f2b00e8039af6ea8887ec124dad8daf19da6fbf1):


```
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/db.go:901
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:105
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:69
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/retry.go:213
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:75
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:57
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:40
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:104
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2383/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:176
        	Error:      	Received unexpected error:
        	            	protected timestamp record does not exist
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts.init
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/protectedts.go:31
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6340
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.main
        	            	  | 	GOROOT/src/runtime/proc.go:233
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) protected timestamp record does not exist
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestJobsProtectedTimestamp/system-tenant
I221210 07:00:10.436401 14 testutils/testcluster/testcluster.go:137  [-] 232  TestCluster quiescing nodes
W221210 07:00:10.436662 3989 kv/kvserver/intentresolver/intent_resolver.go:799  [-] 233  failed to gc transaction record: could not GC completed transaction anchored at /Table/31/1/1/0: node unavailable; try another peer
W221210 07:00:10.436792 383 jobs/registry.go:953  [n1] 234  canceling all adopted jobs due to stopper quiescing
I221210 07:00:10.436948 377 sql/stats/automatic_stats.go:509  [n1] 235  quiescing auto stats refresher
W221210 07:00:10.438276 1609 ccl/kvccl/kvtenantccl/connector.go:234  [nsql1,tenant-connector] 236  error consuming GossipSubscription RPC: grpc: context canceled [code 1/Canceled]
E221210 07:00:10.438587 2030 kv/kvserver/protectedts/ptreconcile/reconciler.go:155  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821117614200750081] 237  failed to reconcile protected timestamp with id c403f4af-a2b3-4747-8826-54979bab1e5c: load-job-query: context canceled
W221210 07:00:10.438621 1714 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 238  exiting heartbeat loop
W221210 07:00:10.438785 1714 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 239  exiting heartbeat loop
W221210 07:00:10.439045 1610 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 240  error consuming TenantSettings RPC: grpc: context canceled [code 1/Canceled]
W221210 07:00:10.439484 350 sql/sqlliveness/slinstance/slinstance.go:312  [n1] 241  exiting heartbeat loop
W221210 07:00:10.439588 350 sql/sqlliveness/slinstance/slinstance.go:297  [n1] 242  exiting heartbeat loop
I221210 07:00:10.439761 615 jobs/registry.go:1373  [n1] 243  AUTO SPAN CONFIG RECONCILIATION job 821117612093079553: stepping through state succeeded
E221210 07:00:10.439902 615 jobs/registry.go:825  [n1] 244  error getting live session: node unavailable; try another peer
I221210 07:00:10.440163 1951 jobs/registry.go:1373  [nsql1] 245  AUTO SPAN CONFIG RECONCILIATION job 821117614200750081: stepping through state succeeded
W221210 07:00:10.440184 1778 jobs/registry.go:953  [nsql1] 246  canceling all adopted jobs due to stopper quiescing
E221210 07:00:10.440258 1951 jobs/registry.go:825  [-] 247  error getting live session: node unavailable; try another peer
I221210 07:00:10.440497 1772 sql/stats/automatic_stats.go:509  [nsql1] 248  quiescing auto stats refresher
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.19s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7916923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7916923?buildTab=artifacts#/) on master @ [c050c9b4b57ecf2ceb5d449c31c617fe12c920e0](https://github.com/cockroachdb/cockroach/commits/c050c9b4b57ecf2ceb5d449c31c617fe12c920e0):


```
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/db.go:901
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:105
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:69
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/retry.go:213
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:75
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:57
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/soon.go:40
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:104
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2714/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test_/jobsprotectedtsccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go:176
        	Error:      	Received unexpected error:
        	            	protected timestamp record does not exist
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts.init
        	            	  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/protectedts.go:31
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6340
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.doInit
        	            	  | 	GOROOT/src/runtime/proc.go:6317
        	            	  | runtime.main
        	            	  | 	GOROOT/src/runtime/proc.go:233
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) protected timestamp record does not exist
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestJobsProtectedTimestamp/system-tenant
I221211 06:45:31.448262 26 testutils/testcluster/testcluster.go:137  [-] 235  TestCluster quiescing nodes
W221211 06:45:31.448481 1887 kv/kvserver/intentresolver/intent_resolver.go:799  [-] 236  failed to gc transaction record: could not GC completed transaction anchored at /Tenant/10/Table/37/1/821397850157285377/0: node unavailable; try another peer
I221211 06:45:31.448666 392 sql/stats/automatic_stats.go:509  [n1] 237  quiescing auto stats refresher
W221211 06:45:31.448696 398 jobs/registry.go:953  [n1] 238  canceling all adopted jobs due to stopper quiescing
W221211 06:45:31.448792 189 sql/sqlliveness/slinstance/slinstance.go:312  [n1] 239  exiting heartbeat loop
W221211 06:45:31.448850 189 sql/sqlliveness/slinstance/slinstance.go:297  [n1] 240  exiting heartbeat loop
I221211 06:45:31.449076 1741 sql/stats/automatic_stats.go:509  [nsql1] 241  quiescing auto stats refresher
I221211 06:45:31.449089 620 jobs/registry.go:1373  [n1] 242  AUTO SPAN CONFIG RECONCILIATION job 821397848300781569: stepping through state succeeded
W221211 06:45:31.449138 1747 jobs/registry.go:953  [nsql1] 243  canceling all adopted jobs due to stopper quiescing
E221211 06:45:31.449166 620 jobs/registry.go:825  [n1] 244  error getting live session: node unavailable; try another peer
W221211 06:45:31.449300 1547 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 245  error consuming TenantSettings RPC: grpc: node unavailable; try another peer [code 2/Unknown]
E221211 06:45:31.449629 1942 kv/kvserver/protectedts/ptreconcile/reconciler.go:155  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821397850114162689] 246  failed to reconcile protected timestamp with id 461f27a1-e819-4773-a3f1-dfe42ac3cfe3: load-job-query: context canceled
W221211 06:45:31.449691 1655 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 247  exiting heartbeat loop
W221211 06:45:31.449710 1655 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 248  exiting heartbeat loop
W221211 06:45:31.449656 1546 ccl/kvccl/kvtenantccl/connector.go:234  [nsql1,tenant-connector] 249  error consuming GossipSubscription RPC: grpc: node unavailable; try another peer [code 2/Unknown]
I221211 06:45:31.449751 1866 jobs/registry.go:1373  [nsql1] 250  AUTO SPAN CONFIG RECONCILIATION job 821397850114162689: stepping through state succeeded
E221211 06:45:31.449816 1866 jobs/registry.go:825  [-] 251  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.15s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test.TestJobsProtectedTimestamp [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924144?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924144?buildTab=artifacts#/) on master @ [942a4d468e9c8ad0ef45a7be33f0a326dfb19fef](https://github.com/cockroachdb/cockroach/commits/942a4d468e9c8ad0ef45a7be33f0a326dfb19fef):


```
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:376
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | [...repeated from below...]
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +Wraps: (2) failed to read records
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +Wraps: (3) attached stack trace
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  -- stack trace:
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).execInternal.func1.1
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:847
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next.func1
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:427
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/sql.(*rowsIterator).Next
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/sql/internal.go:434
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).getRecords
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:368
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage.(*storage).GetState
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptstorage/pkg/kv/kvserver/protectedts/ptstorage/storage.go:322
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile.func1
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:124
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn.func1
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:960
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*Txn).exec
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/txn.go:926
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.runTxn
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:959
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).TxnWithAdmissionControl
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:922
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Txn
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:901
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).reconcile
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:122
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).run
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:107
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile.(*Reconciler).StartReconciler.func1
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile/reconciler.go:77
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | runtime.goexit
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +  | 	GOROOT/src/runtime/asm_amd64.s:1594
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +Wraps: (4) protectedts-GetRecords
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +Wraps: (5) context canceled
E221212 06:49:43.899010 1886 kv/kvserver/protectedts/ptreconcile/reconciler.go:128  [nsql1,job=AUTO SPAN CONFIG RECONCILIATION id=821681792553975809] 239 +Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *errors.errorString
W221212 06:49:43.899422 1611 sql/sqlliveness/slinstance/slinstance.go:312  [nsql1] 241  exiting heartbeat loop
W221212 06:49:43.899446 1611 sql/sqlliveness/slinstance/slinstance.go:297  [nsql1] 242  exiting heartbeat loop
W221212 06:49:43.899399 1265 ccl/kvccl/kvtenantccl/connector.go:234  [nsql1,tenant-connector] 243  error consuming GossipSubscription RPC: grpc: node unavailable; try another peer [code 2/Unknown]
I221212 06:49:43.899505 1748 sql/stats/automatic_stats.go:509  [nsql1] 244  quiescing auto stats refresher
W221212 06:49:43.899596 1754 jobs/registry.go:953  [nsql1] 245  canceling all adopted jobs due to stopper quiescing
W221212 06:49:43.899619 1586 ccl/kvccl/kvtenantccl/setting_overrides.go:46  [nsql1,tenant-connector] 246  error consuming TenantSettings RPC: grpc: context canceled [code 1/Canceled]
I221212 06:49:43.899710 1898 jobs/registry.go:1373  [nsql1] 247  AUTO SPAN CONFIG RECONCILIATION job 821681792553975809: stepping through state succeeded
E221212 06:49:43.899791 1898 jobs/registry.go:825  [-] 248  error getting live session: node unavailable; try another peer
    --- FAIL: TestJobsProtectedTimestamp/system-tenant (0.14s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81185 pkg/ccl/jobsccl/jobsprotectedtsccl/jobsprotectedtsccl_test: TestJobsProtectedTimestamp failed [C-test-failure O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestJobsProtectedTimestamp.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/jobsccl/jobsprotectedtsccl/BUILD.bazel
+++ b/pkg/ccl/jobsccl/jobsprotectedtsccl/BUILD.bazel
@@ -25,6 +25,7 @@ go_test(
         "//pkg/security/securitytest",
         "//pkg/security/username",
         "//pkg/server",
+        "//pkg/settings/cluster",
         "//pkg/sql",
         "//pkg/sql/catalog/descpb",
         "//pkg/testutils",

--- a/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go
+++ b/pkg/ccl/jobsccl/jobsprotectedtsccl/jobs_protected_ts_test.go
@@ -25,6 +25,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/scheduledjobs"
 	"github.com/cockroachdb/cockroach/pkg/security/username"
+	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
 	"github.com/cockroachdb/cockroach/pkg/sql"
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
@@ -38,6 +39,30 @@ import (
 	"github.com/gogo/protobuf/types"
 	"github.com/stretchr/testify/require"
 )
+
+// Register a fake job resumer for the test. We don't want the job to do
+// anything.
+func init() {
+	jobs.RegisterConstructor(
+		jobspb.TypeSchemaChangeGC,
+		func(_ *jobs.Job, _ *cluster.Settings) jobs.Resumer {
+			return fakeResumer{}
+		},
+		jobs.UsesTenantCostControl,
+	)
+}
+
+type fakeResumer struct{}
+
+func (f fakeResumer) Resume(ctx context.Context, _ interface{}) error {
+	<-ctx.Done()
+	return ctx.Err()
+}
+
+func (f fakeResumer) OnFailOrCancel(ctx context.Context, _ interface{}, _ error) error {
+	<-ctx.Done()
+	return ctx.Err()
+}
 
 func testJobsProtectedTimestamp(
 	ctx context.Context,

